requirements.txt

```diff
@@ -23,3 +23,4 @@ pmdarima >=1.8.0,<=1.8.3
 black >=21.9b0
 typing_extensions
 colorlog ==6.5.0
+dask==2022.1.0
```

Comment:
This will lead to installation issues, make `requirements_dask.txt` instead and add it there. Or maybe a broad `requirements_parallelsim.txt` where we can have things like dask, ray, modin, etc